ui/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
Remove file

Reply:
removed in https://github.com/AECgeeks/ifc-pipeline-validation/pull/51/commits/c181fbc1ee91f68009e457f01232712646753679